### PR TITLE
Fix attribute errors when scraping and filing.json structure

### LINKED ISSUE #5: Handle attribute error
An attribute error is causing the scrapes to fail, [as seen here](https://github.com/labordata/lm10/actions/runs/10190342230/job/28189873589#step:6:22). We can likely resolve it [similarly to the way we did it for lm20](https://github.com/labordata/lm20/issues/12).

### PR DESCRIPTION
**Pull request by xmedr**
## Overview

Note: The issue here is very similar to the one resolved in [this lm20 pr](https://github.com/labordata/lm20/pull/13). So, much of this solution has been co-opted from that.

The `content-dispositions` when looking for filenames as of late have been either of type `str` or `None`. So the `decode()` method that has been in use has been raising `AttributeErrors`. This branch accounts for the `content-disposition` being a either of those values, and passes the filename found within.

After resolving that error, `jq` had issues parsing out the `files` attributes of objects in `filing.json` since they were nested objects. This branch also flattens out that object with a make command to get the command working.

- Closes #5 

## Testing Instructions

 * Run `make lm20.db` or look at the checks performed on this PR
 * Confirm that everything in the "build database" portion runs without the str/None attribute error
 * Confirm that the data in the db is what we expect

### CODE REVIEW

**Comment by xmedr** [comment]
@fgregg this pr seems to fail because the associated action is trying to push up all of the reports that it found, which locally is about 6gb worth. looking at [past successful actions](https://github.com/labordata/lm10/actions/runs/8699918782), it looks like it hadn't been pushing things before because the `reports` directory hadn't existed. this branch is now set up to have that directory, and is what contains those 6gigs worth of data when this action runs.

im finding out that github has a 2gb limit on pushes, so [this source suggests pushing up in smaller batches](https://stackoverflow.com/a/78393464) to get past that. i have those 6gigs worth of reports locally, so a potential solution could be that i can push them up here in 3 or 4 batches to get things up to date. how does that sound to you, do you think that would work out?

**Comment by xmedr** [comment]
I'll get a change in to prevent this from committing the reports for now

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -54,8 +54,8 @@ jobs:
           asset_path: ./lm10.db.zip # path to archive to upload
           asset_name: lm10.db.zip # name to upload the release as, use $$ to insert date (YYYYMMDD) and 6 letter commit hash
           asset_content_type: application/zip # required by GitHub API
-      - name: Commit changes
-        uses: EndBug/add-and-commit@v7
-        with:
-          message: 'update'
-          add: 'reports/full/*'
+      # - name: Commit changes
```
Comment (fgregg) [inline .github/workflows/main.yml]: let's just delete this.

**Review by fgregg — APPROVED** [review]
looks great! one thing to change here.

File: .github/workflows/main.yml (hunk 1 above)
Comment (xmedr) [reply .github/workflows/main.yml]: got it!

**Pull request merged**